x-pack/plugins/observability_solution/infra/public/plugin.ts

```diff
@@ -132,14 +139,18 @@ export class Plugin implements InfraClientPluginClass {
       messageFields: this.config.sources?.default?.fields?.message,
     });
 
-    const startDep$AndHostViewFlag$ = combineLatest([from(core.getStartServices())]);
+    const startDep$AndAccessibilityFlag$ = combineLatest([from(core.getStartServices())]).pipe(
+      switchMap(([[{ application }]]) =>
```

Comment:
Nice! I think the outer `combineLatest` is just a left-over of a prior change.

```suggestion
    const startDep$AndAccessibilityFlag$ = from(core.getStartServices()).pipe(
      switchMap(([{ application }]) =>
```